config/routes.rb

```diff
@@ -22,6 +22,14 @@
     resources :spaces
     resources :categories
 
+    resources :alerts, :blogs, :blog_posts, :buildings, :collections,
+      :events, :exhibitions, :groups, :finding_aids , :highlights,
+      :people, :policies, :services, :spaces do
+      member do
+        post :revert
```

Comment:
Does this still need to exist? Aren't we just using the edit route now?

Reply:
Revert method removed.